doc/source/serve/architecture.rst

```diff
@@ -26,7 +26,8 @@ There are three kinds of actors that are created to make up a Serve instance:
   responds once they are completed.
 - Worker Replica: Worker replicas actually execute the code in response to a
   request. For example, they may contain an instantiation of an ML model. Each
-  replica processes individual requests or batches of requests from the routers.
+  replica processes individual requests from the routers (they may be batched
+  by the replica using @serve.batch).
```

Comment:
```suggestion
  by the replica using ``@serve.batch``).
```

Sorry I know this is really annoying but can we add a link to the batching section somewhere here?

Reply:
yup, done